functional-samples/ai.gemini-on-device/README.md

```diff
@@ -1,14 +1,20 @@
 # On-device AI with Gemini Nano
 
-This sample demonstrates how to use the experimental Gemini Nano API in Chrome. To learn more about the API and how to sign-up for the preview, head over to [Built-in AI on developer.chrome.com](https://developer.chrome.com/docs/ai/built-in).
+This sample demonstrates how to use the experimental Gemini Nano API available in the context of an origin trial in Chrome with Chrome Extensions. To learn more about the API and how to sign-up for the origin trial, head over to [Built-in AI on developer.chrome.com](https://developer.chrome.com/docs/ai/built-in).
 
 ## Overview
 
-The extension provides a chat interface using the prompt API with Chrome's built-in Gemini Nano model.
+The extension provides a chat interface using the Prompt API with Chrome's built-in Gemini Nano model.
 
 ## Running this extension
 
 1. Clone this repository.
-2. Load this directory in Chrome as an [unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
-3. Click the extension icon.
-4. Interact with the prompt API in the sidebar.
+1. Launch Chrome with the following flags:
```

Comment:
`--enable-features=EnablePromptAPIForExtension` isn't required, right? I'm fairly sure the origin trial enables this for you.

Reply:
Ah, of course. PTAL.